src/app/app.module.ts

```diff
@@ -22,14 +22,18 @@ import {MapPageComponent} from './pages/map-page/map-page.component';
 import {LayersService} from '../services/layers.service';
 import {SupportService} from '../services/support.service';
 import {HttpClientModule} from '@angular/common/http';
-import {AgmCoreModule} from '@agm/core';
+import {AgmCoreModule, GoogleMapsAPIWrapper} from '@agm/core';
 import {MarkerDescriptionComponent} from './pages/map-page/marker-description/marker-description.component';
 import { SendingReportPageComponent } from './pages/sending-report-page/sending-report-page.component';
 import {LoginPageComponent} from './pages/login-page/login-page.component';
 import {AuthServiceConfig, SocialLoginModule} from 'angular-6-social-login';
 import {getAuthServiceConfig} from '../services/auth.service';
 import {AuthService} from '../services/auth.service';
 import {FacebookModule} from "ngx-facebook";
+import {UpdateService} from '../services/update.service';
+import {MatDialogModule} from '@angular/material/dialog';
+import { NewVersionAlertComponent } from './general-components/new-version-alert/new-version-alert.component';
+import {CoreMapContentComponent} from './pages/map-page/core-map/core-map-content.component';
 // import { MapPageComponent } from './pages/map-page/map-page.component';
```

Comment:
starting now, i'll start review for better code design.
for example we'll delete comments, if it is not used, even if you didn't do the comment, we'll start designing better code :)

Reply:
i removed all comments and unused packages

Reply:
and yes, now we have the time and the will to do it!